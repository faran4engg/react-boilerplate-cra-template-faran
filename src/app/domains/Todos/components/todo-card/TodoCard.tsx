// @ts-nocheck
const TodoCard = ({ isLoading, todos }) => (
  <>
    <p>Swag will be here</p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum ducimus
      veniam et accusantium maxime sed dolores repudiandae non voluptatum
      exercitationem tempora suscipit molestiae pariatur, voluptatem velit
      vitae, illo, debitis consequuntur officia maiores nesciunt autem itaque.
      Labore, expedita, recusandae error voluptate hic accusamus harum aliquid
      quam qui iure at vitae, quisquam animi. Voluptas fugiat libero provident
      ratione veritatis accusantium repellendus non minima voluptates, iste ab
      commodi facilis nesciunt placeat repellat id, porro similique, maiores
      illum beatae delectus. Enim a doloremque labore iste aliquid eius
      molestias suscipit consequuntur consequatur esse. Necessitatibus aliquam
      eveniet aspernatur, tenetur voluptates quia ducimus commodi voluptas.
      Quidem, recusandae. Veniam, voluptas deleniti esse id alias iure
      recusandae? Officia cumque facere consectetur iusto! Mollitia similique
      expedita numquam iste ratione quisquam natus illum dolore tempore fugit?
      Ipsam cum natus dolores eveniet itaque assumenda voluptates corporis,
      exercitationem, sunt consectetur modi illo ipsa autem ut sequi! Quasi,
      error non repellat, optio aliquam soluta itaque voluptas inventore nisi
      deserunt quo deleniti repudiandae autem totam accusamus natus aliquid
      quaerat laudantium. Facilis delectus neque adipisci est, itaque sit iste,
      consequuntur voluptates qui provident deleniti similique ullam? Impedit
      quod suscipit ut ipsam minima fugit cum molestias nisi consectetur
      doloribus, accusantium deleniti odio beatae eum recusandae ad eveniet
      culpa assumenda reprehenderit fugiat sint. Minus fugiat nihil nostrum ex
      doloremque unde architecto ipsa laborum atque assumenda officia reiciendis
      omnis voluptatum explicabo rem labore aliquid amet, debitis, ratione, nemo
      doloribus harum illo quibusdam sint? Autem odio voluptatum nostrum,
      recusandae unde ducimus quod doloremque sed deleniti quidem alias incidunt
      nemo aliquid impedit numquam omnis. Saepe, error. Tenetur, porro dolor
      molestias iste nesciunt dicta odio consequatur temporibus nihil ullam quos
      a quisquam. Illum nostrum explicabo commodi dolorum perferendis eos neque
      itaque optio nulla? Adipisci eaque voluptatibus dolores, quasi dolor et
      quae accusantium error consectetur voluptatem animi dignissimos hic
      molestiae dolorem quidem ducimus deserunt laboriosam provident blanditiis,
      excepturi sunt exercitationem. Praesentium perspiciatis voluptas aliquid,
      dolorem cumque quibusdam architecto? Totam doloremque eligendi culpa?
      Laborum, animi placeat ducimus suscipit obcaecati sed repellat esse
      incidunt officiis, reprehenderit nihil possimus earum cum illum est rem
      consequatur voluptates quidem nobis minus veritatis saepe perferendis
      consequuntur velit? Voluptates minus illum sapiente facilis excepturi rem,
      nisi nam aliquid. Necessitatibus mollitia maxime quidem veritatis,
      obcaecati soluta aut voluptatem libero ducimus porro, placeat odit omnis.
      Rem laudantium voluptatem ratione tempore ex dolores aperiam magni
      commodi, voluptate enim, similique adipisci facilis veritatis velit! Quos
      libero, quo modi hic ipsam maxime quia ad ipsum quis velit, doloribus,
      beatae ullam laboriosam optio est? Numquam facilis voluptatem, ratione
      perferendis atque, aut beatae tenetur obcaecati molestias nisi laboriosam,
      debitis doloremque vel esse tempore omnis dicta voluptatum! Sapiente
      voluptatem totam, blanditiis adipisci corrupti praesentium voluptate.
      Dolores, quidem labore! Impedit animi natus exercitationem mollitia magni
      pariatur cum, sapiente aliquid nesciunt, dolorum commodi distinctio
      cupiditate iure consectetur quis! Blanditiis quia, eveniet minus atque
      doloribus sunt beatae? Dolorem, ullam laborum a, est omnis architecto
      sequi, possimus modi atque blanditiis non! Reiciendis laboriosam
      accusantium culpa tenetur vitae. Magnam enim facilis assumenda quidem
      voluptatem iste provident tenetur rerum expedita dolorem? Maxime fugiat
      vel dolorum assumenda exercitationem expedita quos odit. Corrupti tempore
      autem esse consequuntur soluta molestias illum possimus quia voluptate,
      laudantium nihil culpa error ea dolore suscipit consequatur? Recusandae
      sunt maiores deleniti atque rem aliquam placeat molestias laudantium, quis
      consequuntur ipsa provident consectetur soluta nam quas molestiae ratione
      est fugit fugiat eum doloremque? Nisi saepe in adipisci similique
      inventore temporibus, omnis consequuntur repellat reprehenderit atque eius
      id at rem porro velit, enim facere harum, veniam nostrum odio libero
      possimus officia autem? Sapiente est a dolores laboriosam corporis, optio
      obcaecati sint quaerat officia itaque recusandae repudiandae quas ea et
      blanditiis! Et nobis nemo eaque possimus cumque! Nesciunt, maxime cum.
      Cumque praesentium optio deserunt qui quisquam quas sunt voluptates dolor
      rem repudiandae ex molestias, sint labore nihil expedita sed minus
      mollitia. Omnis soluta eum sint natus officia saepe rem iste quaerat
      officiis quis nobis earum sed corporis modi aperiam, optio deserunt beatae
      dicta sit quibusdam, mollitia, exercitationem explicabo amet molestiae! Ab
      tenetur similique impedit facilis iusto maiores fugiat saepe, reiciendis
      quod tempore nostrum ratione beatae repudiandae eveniet, rerum qui non
      quaerat inventore modi dolorem? Libero porro neque dolores quis
      perspiciatis, consequatur vel at sequi magnam tempora quidem, minus nemo
      nam iusto adipisci officia velit ipsa facere nisi. Cupiditate nam facere
      porro reiciendis aspernatur quis praesentium, id sequi quae ut dicta,
      voluptatem vel eos qui dolorum esse architecto saepe ratione, molestias
      quisquam incidunt iusto veritatis numquam? Impedit praesentium doloremque
      optio nostrum eius voluptas, quo exercitationem id ratione natus sapiente
      incidunt laborum voluptatum aliquam necessitatibus harum neque laudantium
      et a? Doloremque, ducimus fuga nemo nobis soluta assumenda asperiores rem
      obcaecati autem expedita error ex odio voluptatem nulla dicta adipisci
      sint dolore illum consequuntur laudantium provident tenetur perferendis.
      Ut ratione natus eius perspiciatis pariatur magni doloribus odio possimus
      neque accusantium corporis error, architecto tempora cumque nostrum alias
      praesentium enim iure voluptates beatae sapiente at fugiat esse
      temporibus? Deleniti et, porro, aliquid neque quasi minus ab asperiores
      voluptate non mollitia harum quas at quisquam id sit quidem, animi
      obcaecati aspernatur. Sed quidem atque hic temporibus et ullam
      consequatur, quasi sequi repellendus vitae nesciunt. Ratione ipsa vel
      quisquam, laboriosam nulla tenetur delectus minima esse quod. Totam
      repellat facilis repellendus similique vitae ipsa itaque impedit quas
      minus neque est, optio dicta nemo commodi blanditiis culpa incidunt
      accusamus illum voluptatibus omnis. Culpa amet saepe similique repellat
      officiis. Ipsa totam perspiciatis reiciendis adipisci hic omnis illum
      quibusdam libero laborum. Laborum non provident in voluptatem culpa id,
      est mollitia repellendus officia quas libero doloribus iusto, ut totam
      tenetur. Exercitationem error ad id harum est itaque, commodi aliquid
      dolorum minima corrupti in excepturi labore quis, ipsam veniam voluptates!
      Amet dicta voluptate aspernatur sed? Veritatis accusantium nam pariatur
      consectetur neque modi voluptatem sequi totam! Vel quae est iure hic!
      Provident explicabo fugiat saepe, doloremque omnis quas debitis laboriosam
      officia maiores minima, pariatur, vitae placeat dolorum voluptas sequi
      beatae laudantium repellendus corrupti qui facere ea illo. Repellendus
      asperiores modi, ea corrupti pariatur tempore alias, commodi excepturi,
      quidem distinctio dolores ullam dignissimos. Quae esse voluptas quaerat
      quam incidunt.
    </p>
  </>
);

export default TodoCard;
