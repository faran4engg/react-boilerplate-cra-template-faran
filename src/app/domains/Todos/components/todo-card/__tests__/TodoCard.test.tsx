import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import TodoCard from '../TodoCard';

const renderer = createRenderer();

describe('<TodoCard />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<TodoCard isLoading={false} todos={[]} />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
