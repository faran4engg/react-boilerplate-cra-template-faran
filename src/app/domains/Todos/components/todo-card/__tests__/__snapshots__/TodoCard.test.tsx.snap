// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoCard /> should render and match the snapshot 1`] = `
<React.Fragment>
  <h1
    className="text-6xl"
  >
    Heading 1
  </h1>
  <h2
    className="text-5xl"
  >
    Heading 1
  </h2>
  <h3
    className="text-4xl"
  >
    Heading 1
  </h3>
  <h4
    className="text-3xl"
  >
    Heading 1
  </h4>
  <h5
    className="text-2xl"
  >
    Heading 1
  </h5>
  <h6
    className=""
  >
    Heading 1
  </h6>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil velit esse unde repudiandae exercitationem impedit nostrum obcaecati blanditiis cupiditate eveniet quaerat ipsum a eaque doloremque aliquid, deleniti rem iste porro consequuntur possimus reiciendis beatae. Recusandae animi vero perferendis temporibus enim numquam sequi veniam cumque ea beatae laborum aut natus nostrum libero sint, eligendi quas sunt iure accusantium tempora explicabo? Fuga, necessitatibus aut. Optio, alias veniam. Minus cupiditate neque accusantium nisi magnam similique labore optio, id atque. Rem iste nesciunt soluta repellat deserunt sed tempore tenetur, delectus minus? Autem numquam modi esse quos. Illo nisi magni ut. Doloremque culpa voluptate officiis cumque magnam odit eaque? Voluptatum quibusdam, quae voluptatibus excepturi dolorem facere eius iste tenetur perferendis voluptates consequuntur! Eos iusto eius minus fugit iure est et! Iste ipsum ipsam error itaque molestias, vitae corporis autem velit voluptates excepturi quisquam ut delectus cum repudiandae corrupti quis quibusdam porro eos odio aperiam minima illum beatae iusto? Delectus eaque dolores esse deserunt recusandae suscipit quae odio provident minus enim quidem accusamus nihil repudiandae eligendi laudantium adipisci nostrum, quis ipsa architecto voluptatum pariatur nulla porro impedit! Dolorem nihil laudantium enim ullam inventore perspiciatis amet tenetur provident expedita earum fugiat porro libero a modi consequatur fugit dolorum ab, sequi at id facilis, voluptatibus similique commodi quia. Accusamus, ullam! Necessitatibus commodi optio aliquam hic unde! Iure dolorem laudantium id sequi tenetur quia inventore voluptate similique quos laboriosam deserunt, vel sit hic, rerum molestiae omnis perspiciatis repellat maiores aliquam mollitia laborum incidunt dicta modi. Explicabo laborum adipisci consequatur, mollitia quod exercitationem consequuntur delectus libero animi unde assumenda deleniti obcaecati? Vitae nam officia, enim molestiae eligendi voluptate quae quidem, eius temporibus tenetur porro ducimus voluptatibus ex laboriosam impedit omnis excepturi ratione minus recusandae eveniet mollitia distinctio. Officia quas voluptate cum pariatur accusamus. Pariatur earum obcaecati voluptatibus, distinctio cumque tenetur quo similique veniam quam ipsam quod quisquam dignissimos odit necessitatibus voluptatum beatae quibusdam eos voluptas a nam neque! Tenetur minus expedita nam ducimus dolorem esse ipsam fugit, voluptatum dolore praesentium similique dignissimos ullam asperiores dolorum perspiciatis eum dolores consequuntur modi sunt molestias at, quasi et a. Maxime sint nesciunt reiciendis, eveniet ipsum fugit ducimus ipsam unde facere alias temporibus totam excepturi, harum tempora qui, labore ad! Tenetur temporibus incidunt aliquam molestias a architecto, illo optio maiores impedit quas eaque blanditiis placeat aspernatur doloremque praesentium numquam nostrum voluptatibus! Commodi ipsam cum ex neque delectus quod sapiente, nemo hic eaque quam nisi ducimus nihil dolorem nam eligendi dolore! Quasi atque veritatis nisi doloribus tempore a minus temporibus aperiam dolorum ipsa, rerum facilis consequatur hic fugit ipsam eveniet omnis, culpa modi eius, exercitationem soluta ex provident voluptatum quis? Consequatur, pariatur! Commodi repudiandae consectetur veritatis, fugit mollitia alias consequuntur quaerat? Perspiciatis quisquam dolor quae doloremque quis suscipit officia tenetur fugit nobis vel nostrum quo a officiis, sed illum aliquam nam nesciunt. Expedita veniam alias eveniet molestias tenetur numquam deleniti, quibusdam rerum ab rem at soluta incidunt reiciendis architecto quos perspiciatis, labore eos adipisci nam recusandae. Nemo, nulla sint ex neque est molestias iste enim magnam? Voluptate quia, fugiat sapiente, expedita commodi officia debitis placeat soluta, perspiciatis magnam facere quae cupiditate tempora ut praesentium laboriosam ipsum vero sit! Iusto non nesciunt corporis necessitatibus repellendus quae voluptatem, doloribus doloremque error vitae ipsum. Ut maxime nisi rem mollitia impedit voluptatum repellat, molestiae ad, error aperiam enim nesciunt voluptate non assumenda dolor ea ducimus? Voluptatibus nostrum quod beatae atque, quae ducimus illo asperiores officia accusamus dolor impedit ipsam consequatur vel quis vitae libero iure necessitatibus ex. Asperiores, sunt nam incidunt iure libero error quaerat consequuntur suscipit assumenda eligendi ipsam sint impedit amet dicta quo aspernatur ratione? Veritatis, vel odio ratione facilis fugiat nihil iste mollitia nesciunt porro, quidem vitae fugit, labore reprehenderit alias ut ullam illo qui voluptates optio ipsum atque aliquid ab voluptatem. Aliquid deleniti, expedita voluptatibus aut ratione placeat atque suscipit, quisquam quas fugiat veniam? Vitae expedita impedit deserunt blanditiis? Ex voluptates provident neque id voluptatum, nihil harum, rem eligendi sequi earum ipsa possimus minus veniam voluptatibus? Iusto aspernatur quam quae ex tenetur ipsum voluptatum vero illo dolores corrupti molestiae maiores, commodi minus? Libero deleniti, est maxime debitis, fugit deserunt recusandae dolor necessitatibus explicabo aut molestiae repellat at doloremque provident ullam exercitationem blanditiis rem dolorem non ducimus, vitae enim ipsam ea corrupti. Reiciendis expedita, placeat ex earum repellat eaque eligendi provident nostrum illum, iure voluptatum! Ex voluptates culpa sunt molestias, qui autem. Fugiat omnis cum quibusdam temporibus natus in sunt blanditiis! Laboriosam, facere! Ab atque minima iure consectetur, id commodi? Laboriosam ad molestias reprehenderit a cumque sint saepe, maxime mollitia dolore quam in earum magni eum aut vel omnis temporibus itaque! Recusandae illum, cupiditate, magnam quod asperiores nisi quas provident, qui ipsam autem repellat! Reprehenderit eos, dicta aut necessitatibus voluptate eligendi itaque expedita in neque? Eligendi earum, illo ex magnam quis est? Facere laudantium recusandae provident pariatur odit dignissimos quia voluptatum repellendus accusamus eos nobis exercitationem repellat perspiciatis eum ad voluptatibus consectetur quos unde vitae, officiis maxime ratione, saepe quidem nisi. Doloribus facilis alias molestiae aut culpa, iusto esse quas incidunt sunt nisi quasi explicabo, omnis neque quibusdam voluptatem odit nulla? Illum assumenda quaerat facere! Quia, ab ut tempore debitis delectus, architecto dolor eaque libero eius quaerat impedit. Pariatur quae accusantium velit distinctio numquam laborum quibusdam ipsum. Pariatur ab ipsum reprehenderit dicta est, blanditiis possimus deserunt laboriosam placeat a, illo deleniti commodi expedita, cum et labore? Odit magnam aperiam, labore accusantium soluta et fugit maiores aut neque quos non? Rem illo totam deleniti exercitationem pariatur vel, aperiam, unde aut inventore adipisci repudiandae quos dicta minus voluptatum dolorem itaque fugit. Facere nulla, atque, tempora nobis amet ipsa eaque quia molestias odio fugiat molestiae officia officiis! Quas, minus consequuntur? Vero ullam sequi veniam, consequuntur quis voluptas laudantium eveniet quod repellat saepe aliquid, rerum, molestias totam repellendus omnis nihil ipsum quae quasi unde tempore hic a in optio. Dicta, iste placeat unde itaque deleniti eligendi eius quisquam, quis sapiente, numquam veritatis obcaecati consequatur incidunt at sed ab expedita neque. Accusamus nesciunt nisi iste quae, dicta dolores temporibus provident rerum.
  </p>
</React.Fragment>
`;
